docs/products/kafka/kafka-connect/howto/debezium-source-connector-pg.rst

```diff
@@ -0,0 +1,108 @@
+Create a Debezium source connector for PostgreSQL
+=================================================
+
+The Debezium source connector extracts the changes committed to the transaction log in a relational database, such as PostgreSQL, and provides them in a standard format into an Apache Kafka topic where can be transformed and read by multiple consumers. 
```

Comment:
This sentence feels a bit long, but I really can't think of how to improve it.

I think `where can be transformed` should probably be `where they can be transformed`.

Would `writes them to an Apache Kafka topic in a standard format` work, instead of `provides them in a standard format into an Apache Kafka topic`? (I don't know what the standard "verb" is for getting stuff into Kafka.)

Reply:
Good suggestion, modified!